recipes/great-expectations/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "great-expectations" %}
```

Comment:
The name on PyPi is "great_expectations". I'm not sure if makes sure to keep it for consistency, even if a dash would have been better. For instance `memory_profiler` on PyPi is still `memory_profiler` on conda..

Reply:
@rth, 
I added the upstream developer as a maintainer and it looks like he is fine either way. Feel free to change it when you merge the PR if you still feel strongly about it.